File: Notification System/src/SMSNotifier.cpp
#include "SMSNotifier.hpp"
#include <iostream>

void SMSNotifier::sendNotification(const std::string& sensorId,
                                   const std::string& message,
                                   const std::string& timestamp) {
    std::cout << "[SMS] " << timestamp
              << " | Capteur: " << sensorId
              << " | Message: " << message << std::endl;
    // TODO: implémenter envoi réel via API SMS
}

File: Notification System/notifiers/sms_notifier.py
from .notifier import Notifier


class SMSNotifier(Notifier):

    def send_notification(self, sensor_id: str, message: str, timestamp: str):
        print(f"[SMS] {timestamp} | Capteur: {sensor_id} | Message: {message}")
        # TODO: implémenter envoi réel via API SMS
